README.md

```diff
@@ -185,45 +194,313 @@ interface LookerEmbedUser {
+          api_token,
+          navigation_token,
+          session_reference_token: session_reference_token || '',
+        },
+        {
+          headers: {
+            'User-Agent': userAgent,
+          },
+        }
+      )
+    )
+    const cacheKey = `${user.external_user_id}/${userAgent}`
+    embedSessions[cacheKey] = response
+    return {
+      api_token: response.api_token,
+      api_token_ttl: response.api_token_ttl,
+      navigation_token: response.navigation_token,
+      navigation_token_ttl: response.navigation_token_ttl,
+      session_reference_token_ttl: response.session_reference_token_ttl,
+    }
+  } catch (error) {
+    console.error('embed session generate tokens failed', { error })
+    throw error
+  }
+}
+```
+
+### Generate token frontend process
+
+When the Looker EmbedSDK is initialized for cookieless embed it requires a simple callback to call the backend process to generate new tokens.
+
+```javascript
+const generateEmbedTokensCallback = async () => {
+  const resp = await fetch('/generate-embed-tokens')
+  if (!resp.ok) {
+    console.error('generate-embed-tokens failed', { resp })
+    throw new Error(
+      `generate-embed-tokens failed: ${resp.status} ${resp.statusText}`
+    )
+  }
+  return await resp.json()
+}
+```
+
+### Initializing the Looker SDK in the frontend
+
+Cookieless embed is initialized by calling `LookerEmbedSDK.initCookieless` passing in the Looker host value and the two callbacks described previously. Once a Looker embed IFRAME is created it will communicate with the Embed SDK running in the host application and use the callbacks appropriately.
+
+```javascript
+
+const acquireEmbedSessionCallback = async () => {
+  . . .
+}
+
+const generateEmbedTokensCallback = async () => {
+  . . .
+}
+
+LookerEmbedSDK.initCookieless(
+  'looker.example.com',
+  acquireEmbedSessionCallback,
+  generateEmbedTokensCallback
+)
+```
+
+### Loading
+
 ## Demo
 
 A simple demo is provided in the `/demo` directory that uses a basic JS frontend and a Python backend. The example backend `demo.py` uses the Looker API to create a signed URL. The example backend `demo_self_signed.py` uses the embed secret and a helper function to sign the URL. The instructions below are for the example using the Looker API.
 
 ### Step 1 - Enable Embedding in your Looker instance
 
-(This is documented in more detail [here](https://docs.looker.com/r/sdk/sso-embed))
+Enabling SSO embedding is documented in more detail [here](https://docs.looker.com/r/sdk/sso-embed).
+Enabling cookieless embedding is documented in more detail [here](https://cloud.google.com/looker/docs/r/sdk/cookieless-embed).
+
+- Navigate to Admin > _Platform_ Embed on your Looker instance. This requires Admin privileges.
+- The demo server runs by default at [http://localhost:8080](http://localhost:8080). By adding that address to "Embedded Domain Whitelist" you can enabled the demo to receive messages from Looker.
+- Turn on "Embed Authentication"
```

Comment:
I'm going to accept this for now as what you have is currently correct as far as the UI is concerned. This will probably need to change though. Embed SSO Authentication also turns on cookieless embed and cookieless embed is not SSO, at least not as far as I am concerned.